src/full_node/block_header_validation.py

```diff
@@ -707,7 +702,7 @@ async def validate_finished_header_block(
             icc_vdf_output = prev_sb.infused_challenge_vdf_output
 
     # 26. Check challenge chain infusion point VDF
-    if header_block.finished_sub_slots is not None:
+    if header_block.finished_sub_slots is not None and header_block.finished_sub_slots != []:
```

Comment:
When is this hit? This should never be hit

Reply:
Just kidding, you were right. It's not an optional, it just a list which is empty sometimes

Reply:
:)